hack/non-olm-install/install-gitops-operator.sh

```diff
@@ -172,12 +186,23 @@ spec:
         - name: GITOPS_CONSOLE_PLUGIN_IMAGE
           value: ${GITOPS_CONSOLE_PLUGIN_IMAGE}
         - name: KAM_IMAGE
-          value: ${KAM_IMAGE}" > ${TEMP_DIR}/env-overrides.yaml
+          value: ${KAM_IMAGE}
+        - name: ARGOCD_CLUSTER_CONFIG_NAMESPACES
+          value: \"${ARGOCD_CLUSTER_CONFIG_NAMESPACES}\"
+        - name: CONTROLLER_CLUSTER_ROLE
+          value: \"${CONTROLLER_CLUSTER_ROLE}\"
+        - name: DISABLE_DEFAULT_ARGOCD_INSTANCE
+          value: \"${DISABLE_DEFAULT_ARGOCD_INSTANCE}\"
+        - name: DISABLE_DEX
+          value: \"${DISABLE_DEX}\"
```

Comment:
here as well

Reply:
Removed all instances of `DISABLE_DEX` env override.